<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Minecraft Server Checker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Minecraft.png</normaloff>Minecraft.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(23, 23, 23);</string>
  </property>
  <widget class="QFrame" name="MainFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);
font: 8pt &quot;MS Shell Dlg 2&quot;;

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="Server_infos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 59, 59);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="ServerIP_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>1</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Server IP:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ServerIP_display">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 230, 230);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Check_button">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40, 40, 40);</string>
     </property>
     <property name="text">
      <string>Check</string>
     </property>
    </widget>
    <widget class="QLabel" name="Ping_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ping:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Ping_display">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 230, 230);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="ServerSoftware_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Server Software:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ServerSoftware_display">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 230, 230);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Status_text">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Status_display">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 230, 230);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Server_players">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 59, 59);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Players_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Players :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Players_display">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 230, 230);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="MinecraftServerChecker">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Minecraft Server Checker</string>
    </property>
   </widget>
   <widget class="QLabel" name="MinecraftLogo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
